### DISCUSSION TITLE
Deprecate support for Python 3.6

### DISCUSSION THREAD

**Discussion by madpah** [CycloneDX/cyclonedx-python-lib#216] [Polls]
Python 3.6 is reached its end of life as of December 2021.

Questions for open discussion:
1. Should `cyclonedx-python-lib` continue to provide support for Python 3.6?
2. If Yes to (1), for how long should we look to maintain support for Python 3.6?

FYI: @jkowalleck

**Comment by jkowalleck** [comment]
my point of view: drop  support or outdated python versions.
whoever needs support for old python might not use latest version of CycloneDX-py ...

**Comment by madpah** [comment]
Given we've had no objections, we will be moving this deprecation into the forthcoming `3.0.x` release.

**Comment by hakandilek** [comment]
#325 is for dropping 3.6 on dev/3.0.0 branch
